src/components/reactTasks/comment.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react'
+
+export default class Comment extends React.Component{
+
+  constructor(){
+    super()
+    this.state = {comments: ['Brilliant!', 'Terrible']}
```

Comment:
You should declare `value` as an empty string here, that way you know straight away that you will be using `value`